lib/user_interface.rb

```diff
@@ -0,0 +1,85 @@
+require_relative './game'
+
+class UserInterface
+  def initialize(io)
```

Comment:
this could be ```def initialize(io = Kernel)``` so that you don't have to call Kernel when you want to run this on line 85